src/utils/findDeskproTag.ts

```diff
@@ -0,0 +1,13 @@
+import { find, toLower } from "lodash-es";
+import { DESKPRO_TAG } from "../constants";
+import type { IssueTag } from "../services/space/types";
+
+const findDeskproTag = (tags: IssueTag[]): IssueTag|void => {
+  if (!Array.isArray(tags)) {
+    return;
+  }
+
+  return find(tags, ({ name }) => toLower(name) === toLower(DESKPRO_TAG.name));
```

Comment:
Avoid lodash here, `toLower` and `find` have native TS alternatives.

Reply:
I changed it like this, because we are looking for the tag that comes from the API
<img width="503" alt="Screenshot 2024-08-20 at 13 04 38" src="https://github.com/user-attachments/assets/f0b9fea4-95cb-4ff6-a390-ec4ff4f20f78">